utils/check_availability.js

```diff
@@ -97,7 +104,7 @@ function main() {
       case 'f':
         break;
       case 'r':
-        checkRollCandidate();
+        checkRollCandidate(args[1]);
```

Comment:
Let's assert that `args[1]` is one of the supported values here, and error out otherwise

Reply:
```js
const supportedChannels = new Map(['stable', 'beta', 'dev']);
```